spec/models/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user) { User.create(name: 'Alice') }
+  let(:post) { user.posts.create(title: 'Sample Post') }
+
+  describe 'validations' do
+    it 'should be valid with valid attributes' do
+      like = Like.create(user:, post_id: post.id)
+      expect(like).to be_valid
+    end
+  end
+
+  describe 'callbacks' do
+    it 'increments the post\'s like_counter after save' do
+      user = create(:user)
+      post = create(:post)
```

Comment:
- [ ] I got the error while running the like_spec.rb.

![Screenshot from 2023-10-01 03-35-52](https://github.com/supreeti/BlogApp-ROR/assets/26767383/cc088987-54c7-4bd6-8946-15472b934bb2)

It seems there is no create method and the user and post is already created so no need to call here again.